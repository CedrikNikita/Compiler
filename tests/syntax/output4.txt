type int
type char
type double
int main
    3.1
