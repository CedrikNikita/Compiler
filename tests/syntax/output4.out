wonderful
wadsd
what
LOL
type int
type char
type double
int main
    3.1
